### PR TITLE
unite poll defaults

### PR DESCRIPTION
**Pull request by bcoca**
ensure adhoc and play tasks use same default setting.
##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

playbook

### CODE REVIEW

File: lib/ansible/playbook/task.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ class Task(Base, Conditional, Taggable, CollectionSearch):
     _loop = FieldAttribute()
     _loop_control = FieldAttribute(isa='class', class_type=LoopControl, inherit=False)
     _notify = FieldAttribute(isa='list')
-    _poll = FieldAttribute(isa='int', default=10)
+    _poll = FieldAttribute(isa='int', default=C.DEFAULT_POLL_INTERVAL)
```
Comment (gundalow) [inline lib/ansible/playbook/task.py:82]: Will anyone mind/care/notice that this has changed from 10 to 15 seconds?

File: changelogs/fragments/align_poll_defaults.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+    - Use async poll default setting for play tasks also, previouslly in only affected adhoc ansible.
```
Comment (gundalow) [inline changelogs/fragments/align_poll_defaults.yml:2]:
```suggestion
    - Use async poll default setting (DEFAULT_POLL_INTERVAL) for play tasks. Previously DEFAULT_POLL_INTERVAL was only used for adhoc ansible.
```

File: lib/ansible/playbook/task.py (hunk 1 above)
Comment (bcoca) [reply lib/ansible/playbook/task.py:82]: i doubt it, but its in clog now and default is suddenly 'globally configurable', before they had to change it on EACH task.

Comment (gundalow) [reply lib/ansible/playbook/task.py:82]: I think it's fine, just wanted to ask the question.

**Review by gundalow — APPROVED** [review]

**Pull request merged**